src/main/java/ch/njol/skript/test/runner/EffAssert.java

```diff
@@ -75,8 +96,8 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 		} finally {
 			logHandler.stop();
 		}
-		
-		return condition != null;
+
+		return (condition != null) && canInit;
```

Comment:
```suggestion
		return condition != null && canInit;
```

Reply:
I know logically these are the same, but I like having the () to make it abundantly clear that it's not `condition != (null && canInit)`.

Reply:
Ah awwight, fair enough